components/TextFields/src/MDCTextInputControllerFullWidth.m

```diff
@@ -504,15 +507,17 @@ - (UIColor *)inlinePlaceholderColor {
 
 + (UIColor *)inlinePlaceholderColorDefault {
   if (!_inlinePlaceholderColorDefault) {
-    _inlinePlaceholderColorDefault = MDCTextInputInlinePlaceholderTextColorDefault();
+    _inlinePlaceholderColorDefault =
+        MDCTextInputControllerFullWidthInlinePlaceholderTextColorDefault();
   }
   return _inlinePlaceholderColorDefault;
 }
 
 + (void)setInlinePlaceholderColorDefault:(UIColor *)inlinePlaceholderColorDefault {
-  _inlinePlaceholderColorDefault = inlinePlaceholderColorDefault
-                                       ? inlinePlaceholderColorDefault
-                                       : MDCTextInputInlinePlaceholderTextColorDefault();
+  _inlinePlaceholderColorDefault =
+      inlinePlaceholderColorDefault
```

Comment:
Super-nit: This can be:

```objectivec
_inlinePlaceholderColorDefault = inlinePlaceholderColorDefault ?: MDCTextInputControllerFullWidthInlinePlaceholderTextColorDefault()
```

Reply:
I went in and undid all Elvises during the purge. I'd pay $.10 an expression for someone to go and put them back into my code.

Reply:
😑